utils/agreement-utils/src/agreement.rs

```diff
@@ -11,15 +17,19 @@ pub const PROPERTY_TAG: &str = "@tag";
 const DEFAULT_FORMAT: &str = "json";
 
 // TODO: Consider different structure:
-//  - 2 fields for parsed properties (demand, offer)
+//  - 2 fields for parsed properties (demand, offer) as ProposalView
 //  - other fields for agreement remain typed.
-// TODO: Move to ya-client to make it available for third party developers.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+// TODO: For compatibility reasons this structure has very similar functions
+//  as ProposalView, but as long as we don't merge them, we need to keep them.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
 pub struct AgreementView {
     pub json: Value,
-    pub agreement_id: String,
+    pub id: String,
 }
 
+pub type OfferView = ProposalView;
+pub type DemandView = ProposalView;
```

Comment:
What are these for? Probably ujnused

Reply:
I think I'm using them in incoming negotiator changes